heat-stack/model/OilPropaneBillRecords.tsx

```diff
@@ -0,0 +1,7 @@
+export interface OilPropaneBillRecords {
+	provider: number
+	periodStartDate: Date
+	periodEndDate: Date
+	gallons: number
+	inclusionOverride: number
```

Comment:
`suggestion`: the database won't have a concept of it, but I might use an [enum ](https://www.typescriptlang.org/docs/handbook/enums.html) here.

Reply:
@alanisaac what should the options be?

Reply:
See https://github.com/codeforboston/home-energy-analysis-tool/pull/133#discussion_r1454343956

Reply:
updated to `inclusionOverride: 'Include' | 'Do not include' | 'Include in other analysis'`